model.go

```diff
@@ -18,14 +18,42 @@ type AuthToken struct {
 
 // Record is the lab test results of the user
 type Record struct {
-	ID             int `json:"id,string,omitempty"`
-	User_id        int `json:"userId,string,omitempty"`
-	Age            int `json:"age,string,omitempty"`
-	Height         int `json:"height,string,omitempty"`
-	Weight         int `json:"weight,string,omitempty"`
-	Cholesterol    int `json:"cholesterol,string,omitempty"`
-	Blood_pressure int `json:"bloodPressure,string,omitempty"`
-	Tip_sent       int `json:"tipSent,string,omitempty"`
+	ID                    int  `json:"id,string,omitempty"`
+	User_id               int  `json:"userId,string,omitempty"`
```

Comment:
while we're doing this, let's follow Go convention and edit some field names. Go uses camel casing instead of underscores and `Id` should be `ID` so let's make the UserID here and throughout

Reply:
I've rewritten all the snake case to camel case.